packages/amplify-migration-codegen-e2e/src/secrets.ts

```diff
@@ -0,0 +1,37 @@
+import { getNpxPath, nspawn as spawn, getSocialProviders } from '@aws-amplify/amplify-e2e-core';
+
+export async function sandboxSecrets(cwd: string, option: string) {
```

Comment:
nice work on keeping this function configurable for `set/unset`! 
nit: rename `sandboxSecrets` to the action its performing on secrets. For e.g. `toggleSandboxSecrets`.